third_party/sl4a/src/main/java/com/google/android/mobly/snippet/facade/FacadeManager.java

```diff
@@ -30,35 +30,24 @@
 
 public class FacadeManager extends SnippetManager {
 
-  private final Context mContext;
   private int mSdkLevel;
 
-  public FacadeManager(int sdkLevel, Context context,
-                       Collection<Class<? extends Snippet>> classList) {
-    super(context, classList);
+  public FacadeManager(int sdkLevel, Collection<Class<? extends Snippet>> classList) {
```

Comment:
Now that we are running under instrumentation, the way to get a context is the same as in any androidTest:
InstrumentationRegistry.getContext() // the test apk's context
InstrumentationRegistry.getTargetContext() // the main apk's context